examples/csv_to_json.py

```diff
@@ -34,7 +28,7 @@ def convert(input, output, **kwargs):
     task_id="convert_to_json",
     provide_context=True,
     python_callable=convert,
-    op_kwargs={'input': "/path/to/my.csvv",
-    'output': "/path/to/my.json"},
+    op_kwargs={'input': "../example.csv",
```

Comment:
And here, I adapted the paths so that if we follow the modified example above, it should work right out of the box.